### PR TITLE
chore: fix ci

### PR DESCRIPTION
**Pull request by morenol**

### CODE REVIEW

**Comment by morenol** [comment]
@sehz could we add branch protection to `main` so we cannot push directly there and cannot merge if CI is not green?

**Comment by sehz** [comment]
Branch protection for main is added.   Need to add "Done" status in the workflow

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -7,6 +7,8 @@ concurrency:
 on:
   merge_group:
   pull_request:
+  push:
```
Comment (sehz) [inline .github/workflows/ci.yml:10]: is there reason to run again in push?

**Review by sehz — CHANGES_REQUESTED** [review]
disable push on main

File: .github/workflows/ci.yml (hunk 1 above)
Comment (morenol) [reply .github/workflows/ci.yml]: no, removed

**Review by sehz — APPROVED** [review]

**Pull request merged**